tasks.html

```diff
@@ -1,59 +1,57 @@
-        <h3>Form Tags</h3>
-        <p>Use of text,pass,radio buttons, placeholder,checkbox,textarea,select box, datalist, file, fieldset, legend, email, submit button, tel</p>
-        <form>
-            <fieldset>
-                <legend>Enter Personal Details</legend>
-            <label>Name</label> <input type="text" name="fname" placeholder="Enter your name"><br><br>
-            <label>Password</label> <input type="password" name="pass"><br><br>
-            <label>Gender</label> <input type="radio" name="gender"><label>Male</label> <input type="radio" name="gender"><label>Female</label><br><br>           
-            <label>Hobbies</label> <input type="checkbox" name="hobby"><label>Music</label> <input type="checkbox" name="hobby"><label>Dancing</label><br><br>
-            <label>Message</label> <textarea placeholder="Enter Description"></textarea><br><br>
-            <label>Country</label> <select>
-                <option selected>India</option>
-                <option>America</option>
-                <option>California</option>
-            </select><br><br>
-            <label>Cars</label> <input list="cars">
-                                <datalist id="cars">
-                                    <option value="Innova">
-                                    <option value="Swift">
-                                    <option value="Kwid">
-                                </datalist><br><br>
-            <label>Upload file</label> <input type="file"><br><br>
-            <label>Email</label> <input type="email"><br><br>
-            <label>Contact No.</label> <input type="tel" /><br><br>
-            <input type="submit" />
-
-
-        </fieldset>
-        </form>
-    </body>
-</html>
\ No newline at end of file
+  <head>
+    <title>Task-1 at Compro</title>
+  </head>
+  <body>
+    <h2 align="center">Task-1 at Compro</h2>
+    <h3>Anchor tag</h3>
+    <a href="mailto:xyz@gmail.com?cc=admin@gmail.com&subject=Test Mail">Contact Us</a>
+    <h3>Table Demonstration</h3>
+    <p>Use of cellspacing, cellpadding, colspan, width, th, tr, td</p>
+    <table border="1" width="400px" cellspacing="2px" cellpadding="10px">
+      <tr>
+        <th>Name</th>
+        <th colspan="2">Designation & company</th>
+      </tr>
+      <tr>
+        <td>Sona Verma</td>
+        <td>ASE</td>
+        <td>Compro</td>
+      </tr>
+      <tr>
+        <td>XYZ</td>
+        <td>ASE</td>
+        <td>Compro</td>
+      </tr>
+    </table>
+    <h3>Iframe tag</h3>
+    <iframe src="sample.html" width="400px" height="300px"></iframe>
+    <h3>Form Tags</h3>
+    <p>Use of text, pass, radio buttons, placeholder, checkbox, textarea, select box, datalist, file, fieldset, legend, email, submit button, tel</p>
+    <form>
+      <fieldset>
+        <legend>Enter Personal Details</legend>
+        <label>Name</label> <input type="text" name="fname" placeholder="Enter your name"><br><br>
```

Comment:
Improve readability, for instance - 
```
<label>Name</label>
<input type="text" name="fname" placeholder="Enter your name">
```
Avoid use of `br` tag. Instead try to achieve that specific thing from css.

Reply:
Improve readability to all similar instance as above.